webportal/webapp/src/services/cognito.service.ts

```diff
@@ -133,19 +138,39 @@ export class CognitoService {
                 idToken = session.getIdToken().getJwtToken()
             });
         } else {
-            console.log("NO TOKEN")
+            console.log("NO TOKEN - Retrieving new session.");
+            // If no token is available, clear localStorage and attempt to get a new session
+            // Generally, this appears to result in a logout, which seems OK
+            localStorage.clear();
+            var userAuth = new CognitoAuth(this.authData());
+            userAuth.userhandler = {
+                onSuccess: function(result) {
+                },
+                onFailure: function(err) {
+                }
+            };
+            userAuth.getSession();
```

Comment:
This is the only change I made in respect to proxy rollout. Everything else is encompassed by #86